package.json

```diff
@@ -125,7 +125,29 @@
         {
           "command": "tailscale.openAdminConsole",
           "group": "overflow",
-          "when": "view == tailscale-serve-view"
+          "when": "view == tailscale-serve-view || view == tailscale-node-explorer-view"
+        }
+      ],
+      "view/item/context": [
+        {
+          "command": "tailscale.copyIPv4",
+          "when": "view == tailscale-node-explorer-view && viewItem == tailscale-peer-item",
+          "group": "1_peer@1"
+        },
+        {
+          "command": "tailscale.copyIPv6",
+          "when": "view == tailscale-node-explorer-view && viewItem == tailscale-peer-item",
+          "group": "1_peer@1"
+        },
+        {
+          "command": "tailscale.copyHostname",
+          "when": "view == tailscale-node-explorer-view && viewItem == tailscale-peer-item",
+          "group": "1_peer@1"
+        },
+        {
+          "command": "tailscale.ssh.delete",
+          "group": "fileActions@2",
+          "when": "view == tailscale-node-explorer-view && viewItem == file-explorer-item"
```

Comment:
Out of curiosity, are these still needed, now that they are behind a right-click? Or are the commands below enough?

Reply:
They are still needed. You can see them by right-clicking on a node.

Reply:
The `view/item/context` references the command, so both are needed.